dashboard/static/login/index.html

```diff
@@ -1,148 +1,154 @@
-    </div>
-  </footer>
-
-  <script>
-
-    const menu_btn = document.querySelector('.hamburger');
-    const mobile_menu = document.querySelector('.mobile-nav');
-
-
-
-    menu_btn.addEventListener('click', function () {
-      menu_btn.classList.toggle('is-active')
-      mobile_menu.classList.toggle('is-active')
-    })
-
-    document.querySelectorAll('.mobile-navB').forEach(item => {
-      item.addEventListener('click', event => {
-        menu_btn.classList.toggle('is-active')
-        mobile_menu.classList.toggle('is-active')
-      })
-    })
-
-
-    let mybutton = document.getElementById("myBtn");
-
-    const tooltip = document.querySelector(".tooltip");
-    tooltip.parentElement.addEventListener("mouseleave", () => {
-      tooltip.innerText = "Click to copy"
-    });
-    document.getElementById("command").addEventListener("click", (e) => {
-      navigator.clipboard.writeText(e.target.innerText).then(() => {
-        tooltip.innerText = "Copied!";
-      });
-    });
-    var ksi = false;
-    function check(btn) {
-      btn.disabled = true;
-      fetch("/api/login/verify", {
-        method: "POST",
-        headers: {
-          'Accept': 'application/json',
-          'Content-Type': 'application/json'
-        },
-        body: JSON.stringify({ ksi })
-      }).then(res => res.json()).then(res => {
-        if (res) {
-          window.location.href = "/dashboard";
-        } else {
-          btn.disabled = false;
-        }
-      });
-    }
-    function start(btn) {
-      const input = btn.parentElement.children[0];
-      const container = btn.parentElement;
-      const uid = input.value;
-      ksi = btn.parentElement.children[1].checked;
-      btn.disabled = true;
-
-      fetch("/api/login", {
-        method: "POST",
-        headers: {
-          'Accept': 'application/json',
-          'Content-Type': 'application/json'
-        },
-        body: JSON.stringify({ userId: uid })
-      }).then(async res => {
-        btn.disabled = false;
-        const r = await res.json();
-        container.style.display = "none";
-        document.getElementById("command").innerText = "%login " + r.id + ";" + r.token;
-        document.getElementById("success").style.display = "block";
-      }).catch(e => {
-        console.error(e);
-        btn.disabled = false;
-      });
-    }
-  </script>
-</body>
-
-</html>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Remix - Login</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+  <link rel="stylesheet" href="../login/style.css">
+  <link rel="icon" href="../assets/icon.png" alt="Remix icon">
+</head>
+
+<body>
+
+  <header>
+    <h1><a href="https://remix.fairuse.org/#Home">Remix</a></h1>
+    <nav class="navigation">
+      <a href="https://remix.fairuse.org/#Home" title="Go to home section">Home</a>

```

Comment:
same thing as in commands; please use relative links 

I can do that if you want :)